src/Apps/Collect2/Routes/Collect/CollectionApp.tsx

```diff
@@ -35,7 +35,7 @@ export class CollectionApp extends Component<CollectionAppProps> {
     }
   }
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
We're starting to get warnings in the console and so I ran the react codemod over the codebase.